### ISSUE TITLE
New versions not published to NPM

### ISSUE THREAD

**Issue by lachlancollins** [oxc-project/eslint-plugin-oxlint#130]
The latest version on NPM is v0.5.0, published >1 month ago:

![image](https://github.com/user-attachments/assets/9154a0c5-8a36-45ff-8a82-15a27a2f9378)

However, 3 versions have been released on GitHub since then, with the latest being v0.7.0:

![image](https://github.com/user-attachments/assets/2425aa8e-234c-4b80-b36c-ea4ed65bb64d)

The latest release CI workflow shows that it is not getting published because the version in `package.json` is not getting bumped by the workflow: https://github.com/oxc-project/eslint-plugin-oxlint/actions/runs/10244585173/job/28337848790

```
Run pnpm -r publish --access public --no-git-checks
There are no new packages that should be published
```

**Comment by Boshen** [comment]
I've yet got the time to automate this, would you like to help out? We need to make a new release after `oxlint` is updated.

It's currently triggered by a tag, we need to change this. https://github.com/oxc-project/eslint-plugin-oxlint/actions/runs/10244585173/workflow

**Comment by Boshen** [comment]
This didn't trigger a publish as well 🤔 https://github.com/oxc-project/eslint-plugin-oxlint/actions/runs/10244585173/job/28337848790

**Comment by lachlancollins** [comment]
> It's currently triggered by a tag, we need to change this. [oxc-project/eslint-plugin-oxlint/actions/runs/10244585173/workflow](https://github.com/oxc-project/eslint-plugin-oxlint/actions/runs/10244585173/workflow)

It looks like the v0.7.0 tag has been generated, but it's not publishing because `package.json` still has `"version": "0.5.0"`.

**Comment by lachlancollins** [comment]
> I've yet got the time to automate this, would you like to help out? We need to make a new release after `oxlint` is updated.

@Boshen I think I can try automating this!

**Comment by Boshen** [comment]
It's finally updated https://www.npmjs.com/package/eslint-plugin-oxlint?activeTab=versions

Thank you for the quick updates.
